examples/pom.xml

```diff
@@ -89,6 +89,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
```

Comment:
I guess you forgot to add `ignite-ml-spark-model-parser` into root `pom.xml`. See https://github.com/apache/ignite/pull/5753/files#diff-600376dffeb79835ede4a0b285078036.